lib/komponent/komponent_helper.rb

```diff
@@ -73,4 +72,22 @@ def render_partial(partial_name, locals = {}, &block)
 
     rendered_partial
   end
+
+  def translate(key, options = {})
```

Comment:
Defining it here will override the `translate` method in all views. I think it should instead be put inside a module and prepended on the component context.

Reply:
Seems this feature is already working on master branch.